src/components/app-content-renderer/config-try-learn-tile.js

```diff
@@ -34,12 +37,19 @@ const iconMapper = {
   unknown: QuestionCircleIcon,
 };
 
-const TileItem = ({
-  icon,
-  title,
-  description,
-  link: { href, title: linkTitle } = {},
-}) => {
+const TileItem = ({ url, ...rest }) => {
+  const [
+    { icon, loaded, title, description, link: { href, title: linkTitle } = {} },
+    setData,
+  ] = useState({ loaded: !url, ...rest.shape });
+  useEffect(() => {
+    if (url) {
+      processRequest({ url, ...rest }).then((data) =>
+        setData({ loaded: true, ...data.response.shape })
+      );
+    }
+  }, []);
```

Comment:
There is `useRequest` hook you should use

Reply:
Ah yes! good point.